### PR TITLE
Add conclusion time to dashboard

### PR DESCRIPTION
**Pull request by OliviaLynn**
Adds conclusion_time for workflows.

While not yet implemented, I added some comments where it seemed a check for is_stale would make sense (and have self-assigned #16)

Screenshot:
Screenshot 2023-12-19 at 11 01 47 AM

### CODE REVIEW

File: src/lf_workflow_dash/github_request.py [hunk 1]
```diff
@@ -25,28 +50,39 @@ def update_workflow_status(workflow_elem, token):
...
                 # try next most recent
                 if len(response_json["workflow_runs"]) > 1:
                     last_run = response_json["workflow_runs"][1]
                     conclusion = last_run["conclusion"]
+                    conclusion_time = get_conclusion_time(last_run)
                 else:
                     conclusion = "pending"
+                    conclusion_time = ""
+
+            # TODO set is_stale
```
Comment (delucchi-cmu) [inline src/lf_workflow_dash/github_request.py:82]: It makes me very sad to see TODOs getting pylint warnings. Opened https://github.com/lincc-frameworks/python-project-template/issues/341 to allow a comment like `# TODO(#16) set is_stale`.

File: templates/dash_template.jinja [hunk 2]
```diff
@@ -9,22 +9,99 @@
...
-        <td class="{{project.build_docs.display_class}}"><a href="{{project.build_docs.workflow_url}}">{{project.build_docs.workflow_status}}</a></td>
-        <td class="{{project.live_build.display_class}}"><a href="{{project.live_build.workflow_url}}">{{project.live_build.workflow_status}}</a></td>
+
+        <!-- Smoke Test -->
+        <td class="{{project.smoke_test.display_class}} align-right">
+                    {{project.smoke_test.workflow_status}}
+        </td>
+        <td class="{{project.smoke_test.display_class}} align-center">
+                {% if project.smoke_test.workflow_status == "success" %}
+                    <i class="fa fa-check-circle"></i>
```
[the remark this answers is not in the document]
Comment (OliviaLynn) [reply templates/dash_template.jinja]: Oh, duh! Thanks, that's much better

File: templates/dash_template.jinja [hunk 3]
```diff
@@ -35,8 +112,8 @@
 
     </table>
     <p>
-    Last Updated {{last_updated}} | 
-    <a href='https://github.com/lincc-frameworks/{{dash_repo}}'><i class=\"fa fa-github\"></i> {{dash_repo}}</a>
+        Last Updated {{last_updated}} | 
+        <a href='https://github.com/lincc-frameworks/{{dash_repo}}'><i class=\"fa fa-github\"></i> {{dash_repo}}</a>
```
[the remark this answers is not in the document]
Comment (OliviaLynn) [reply templates/dash_template.jinja:90]: Added in https://github.com/lincc-frameworks/lf-workflow-dash/pull/22/commits/8ffcb59d6d611cff797511bd25bc5ee53d05f5e0

**Review by delucchi-cmu — APPROVED** [review]
Yay!

**Pull request merged**